pkg/cli/client.go

```diff
@@ -17,9 +17,13 @@ import (
 	"os"
 	"path/filepath"
 	"strconv"
+	"strings"
 	"sync"
 	"time"
 
+	"github.com/sigstore/cosign/pkg/oci/remote"
+	"gopkg.in/resty.v1"
```

Comment:
Try to avoid using resty or other http libs in core zot code. We want to minimize this.